### DISCUSSION TITLE
How to customize the entire head

### DISCUSSION THREAD

**Discussion by woaixiaobo** [eclipse-theia/theia#12922] [General]
How to customize the entire head

**Comment by msujew** [comment]
Hey @woaixiaobo,

with the "head" you mean the menu header? It's created in the [BrowserMainMenuFactory](https://github.com/eclipse-theia/theia/blob/c4eeb2caad88b3004bae6d7928ac6b8a53c71839/packages/core/src/browser/menu/browser-menu-plugin.ts#L45-L46). Please give a bit more context on what you're looking to do, so I can help you better.
